test/templates/common.py

```diff
@@ -0,0 +1,62 @@
+import subprocess
+import sys
+import re
+import os
+
+_me = os.path.basename(__file__)
+v = 0
+
+
+def set_basename(file):
+    global _me
+    _me = file
+
+
+def error(msg):
+    """
+    Print nicely-formatted error message and exit.
+    """
+    sys.stderr.write(f'{_me}: error: {msg}\n')
+    sys.exit(1)
+
+
+def run(cmd, stdin=None):
+    if v:
+        print(f"{_me}: running command: {' '.join(cmd)}")
+    with subprocess.Popen(cmd, stdin=stdin, stdout=subprocess.PIPE,
+                          stderr=subprocess.PIPE) as p:
```

Comment:
Could `p` just be `process`?

Reply:
Actually when using `as` variables can be used like this. So up to you. I generally prefer more verbose variable names. This habit is coming from the book called Clean Code for me :)

Reply:
I fixed anyway.